cmd/osquery-perf/agent.go

```diff
@@ -1696,6 +1719,9 @@ func rows(num int, hostUUID string) string {
 }
 
 func main() {
+	go func() {
+		log.Println(http.ListenAndServe("localhost:6060", nil))
```

Comment:
What is this for?

Reply:
See https://pkg.go.dev/net/http/pprof

Basically for debugging goroutines, heap allocation, etc.

Reply:
Ok. Should add a comment in the code.

Reply:
[Add pprof comment](https://github.com/fleetdm/fleet/pull/16746/commits/fb067bcfa539bd63008001a0a1039a3efb81ec41)